src/Dscp/DB/DSL/Interpret/Sqlite3.hs

```diff
@@ -0,0 +1,188 @@
+
+{-# LANGUAGE QuasiQuotes #-}
+
+module Dscp.DB.DSL.Interpret.Sqlite3 where
+
+import Universum
+
+import Control.Monad.Reader
+import Control.Monad.Trans
+
+import qualified Data.Set as Set (Set, singleton, member, empty)
+
+import Database.SQLite.Simple hiding (query)
+import Database.SQLite.Simple.ToField (ToField(..))
+import Database.SQLite.Simple.FromField (FromField(..))
+
+import Text.InterpolatedString.Perl6
```

Comment:
Explicit imports, please. https://github.com/serokell/serokell-util/blob/master/serokell-style.md#imports
The same later.